platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -6,13 +6,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/** Control whether the translation is relative to the map (north) or viewport (screen) */
```

Comment:
Do we need the property's entire documentation? Something like "Circle translate anchor values for use with the `circleTranslateAnchor` property of MGLCirclrStyleLayer" might do the trick.